cmd/kured/main.go

```diff
@@ -141,6 +144,20 @@ func rebootBlocked() bool {
 			log.Warnf("Reboot blocked: %d active alerts: %v", count, alertNames)
 			return true
 		}
+	} else if alertmanagerURL != "" {
+		alertNames, err := alerts.AlertmanagerActiveAlerts(alertmanagerURL, alertFilter)
+		if err != nil {
+			log.Warnf("Reboot blocked: AlertManager query error: %v", err)
+			return true
+		}
+		count := len(alertNames)
```

Comment:
Do you think this section could be merged with the one above where the code is identical?